docs/v1/P4Runtime-Spec.mdk

```diff
@@ -603,11 +603,11 @@ build/embedded-plus-two-remote-controllers.[svg,png] \
 ## Embedded Controller + Two High-Availability Remote Controllers
 
 Figure [#fig-embedded-plus-two-remote-ha-controllers] illustrates a single
-embedded controller plus two remote controllers in an active-standby HA
+embedded controller plus two remote controllers in an active-standby (&ie; primary-backup) HA
 (High-Availability) configuration. Controller #1 is the active controller and is
 in charge of some entities. If it fails, Controller #2 takes over and manages
 the tables formerly owned by Controller #1. The mechanics of HA architectures
-are beyond the scope of this document, but the P4Runtime multi-master
+are beyond the scope of this document, but the P4Runtime active-active primary
```

Comment:
Same as above. I don't think "multi-master" was accurate in the first place, and I am not sure "active-active" applies here since the backup controllers don't do anything until the primary fails. So I would suggest simply "the P4Runtime client arbitration scheme"

Reply:
That make sense, we can also do: "but the P4Runtime role-based client arbitration scheme"